CHANGELOG.md

```diff
@@ -5,16 +5,20 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Added support for PureScript 0.14 and dropped support for all previous versions (#10, #14)
 
 New features:
+- Updated implementation to use Leibniz equality (#11)
 
 Bugfixes:
 
 Other improvements:
+- Migrated CI to GitHub Actions and updated installation instructions to use Spago (#12)
+- Added a changelog and pull request template (#13, #15)
 
```

Comment:
I don't believe it's breaking, but `Coercible` is now a superclass of `TypeEquals`. We added that to fix the tests in #14. So, perhaps a line should be added for that
```
- Fixed tests by making `Coercible` a super class of `TypeEquals` (#14)